apps/test/netsim/NetSimTable.js

```diff
@@ -265,4 +267,57 @@ describe("NetSimTable", function () {
     });
   });
 
+  describe ("incremental update", function () {
+
+    beforeEach(function () {
+      // New table configured for incremental refresh
+      netsimTable = netsimTestUtils.overrideClientApi(
+          new NetSimTable('testShard', 'testTable', {
+            channel: fakeChannel,
+            useIncrementalRefresh: true
+          }));
+
+      // Allow multiple quick reads for testing
+      netsimTable.setRefreshThrottleTime(0);
+
+      // Necessary to re-get apiTable when we recreate netsimTable
+      apiTable = netsimTable.api_.remoteTable;
+    });
+
+    it ("Initially requests from row 1", function () {
+      assertEqual(apiTable.log(), '');
```

Comment:
I believe you want the "expected" value to the go on the left of assertEqual (as seen e.g. on line 304)